src/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -101,6 +101,7 @@ public partial struct ImmutableArray<T> : System.Collections.Generic.ICollection
         public bool IsDefaultOrEmpty { get { throw null; } }
         public bool IsEmpty { get { throw null; } }
         public T this[int index] { get { throw null; } }
+        public ref readonly T ItemRef(int index) { throw null; }
```

Comment:
We cannot add any new APIs to this library targeting netstandard because the API surface is frozen because it shipped inbox on .NET Core 2.0.

Reply:
Any new APIs must be added as platform specific such as netcoreapp2.1.